meta/flag-references/de.md

```diff
@@ -0,0 +1,237 @@
+[flag___]: /wiki/shared/flag/__.gif
+[flag_AD]: /wiki/shared/flag/AD.gif "Andorra"
+[flag_AE]: /wiki/shared/flag/AE.gif "Vereinigte Arabische Emirate"
+[flag_AF]: /wiki/shared/flag/AF.gif "Afghanistan"
+[flag_AG]: /wiki/shared/flag/AG.gif "Antigua und Barbuda"
+[flag_AI]: /wiki/shared/flag/AI.gif "Anguilla"
+[flag_AL]: /wiki/shared/flag/AL.gif "Albanien"
+[flag_AM]: /wiki/shared/flag/AM.gif "Armenien"
+[flag_AN]: /wiki/shared/flag/AN.gif "Niederländische Antillen"
+[flag_AO]: /wiki/shared/flag/AO.gif "Angola"
+[flag_AR]: /wiki/shared/flag/AR.gif "Argentinen"
+[flag_AS]: /wiki/shared/flag/AS.gif "Amerikanisch-Samoa"
+[flag_AT]: /wiki/shared/flag/AT.gif "Österreich"
+[flag_AU]: /wiki/shared/flag/AU.gif "Australien"
+[flag_AW]: /wiki/shared/flag/AW.gif "Aruba"
+[flag_AX]: /wiki/shared/flag/AX.gif "Aland"
+[flag_AZ]: /wiki/shared/flag/AZ.gif "Azerbaidschan"
+[flag_BA]: /wiki/shared/flag/BA.gif "Bosnien und Herzegowina"
+[flag_BB]: /wiki/shared/flag/BB.gif "Barbados"
+[flag_BD]: /wiki/shared/flag/BD.gif "Bangladesch"
+[flag_BE]: /wiki/shared/flag/BE.gif "Belgien"
+[flag_BF]: /wiki/shared/flag/BF.gif "Burkina Faso"
+[flag_BG]: /wiki/shared/flag/BG.gif "Bulgarien"
+[flag_BH]: /wiki/shared/flag/BH.gif "Bahrain"
+[flag_BI]: /wiki/shared/flag/BI.gif "Burundi"
+[flag_BJ]: /wiki/shared/flag/BJ.gif "Benin"
+[flag_BL]: /wiki/shared/flag/BL.gif "Saint-Barthelemy"
+[flag_BM]: /wiki/shared/flag/BM.gif "Bermuda"
+[flag_BN]: /wiki/shared/flag/BN.gif "Brunei"
+[flag_BO]: /wiki/shared/flag/BO.gif "Bolivien"
+[flag_BR]: /wiki/shared/flag/BR.gif "Brasilien"
+[flag_BS]: /wiki/shared/flag/BS.gif "Bahamas"
+[flag_BT]: /wiki/shared/flag/BT.gif "Bhutan"
+[flag_BW]: /wiki/shared/flag/BW.gif "Botswana"
+[flag_BY]: /wiki/shared/flag/BY.gif "Belarus"
+[flag_BZ]: /wiki/shared/flag/BZ.gif "Belize"
+[flag_CA]: /wiki/shared/flag/CA.gif "Kanada"
+[flag_CD]: /wiki/shared/flag/CD.gif "Demokratische Republik Kongo"
+[flag_CF]: /wiki/shared/flag/CF.gif "Zentralafrikanische Republik"
+[flag_CG]: /wiki/shared/flag/CG.gif "Kongo"
+[flag_CH]: /wiki/shared/flag/CH.gif "Schweitz"
```

Comment:
Don't do this to me :')

Reply:
+1

Reply:
~~I just wanted to know if you actually review this 👀 ~~